backend/tracim_backend/tests/functional/test_contents.py

```diff
@@ -4130,6 +4146,20 @@ def test_api__update_thread__err_400__empty_label(self, web_testapp) -> None:
         assert "code" in res.json_body
         assert res.json_body["code"] == ErrorCode.GENERIC_SCHEMA_VALIDATION_ERROR
 
+    def test_api__update_thread__err_400__nasty_mentions(
+        self, web_testapp, html_with_nasty_mention
+    ) -> None:
+        """
+        Update(put) thread
+        """
+        web_testapp.authorization = ("Basic", ("admin@admin.admin", "admin@admin.admin"))
+        params = {"label": "", "raw_content": html_with_nasty_mention}
+        res = web_testapp.put_json("/api/workspaces/2/threads/7", params=params, status=400)
+        # TODO - G.M - 2018-09-10 - Handle by marshmallow schema
+        assert res.json_body
+        assert "code" in res.json_body
+        assert res.json_body["code"] == ErrorCode.GENERIC_SCHEMA_VALIDATION_ERROR
```

Comment:
why `GENERIC_SCHEMA_VALIDATION_ERROR` an not `USER_NOT_MEMBER` ?

Reply:
indeed, that is an oversight